src/tools/loadenv.ts

```diff
@@ -17,7 +17,7 @@ export const kakaoSDKKey = env.REACT_APP_KAKAO_SDK_KEY; // optional
 export const gaTrackingId = env.REACT_APP_GA_TRACKING_ID; // optional
 export const firebaseConfig: Nullable<FirebaseConfig> =
   env.REACT_APP_FIREBASE_CONFIG && JSON.parse(env.REACT_APP_FIREBASE_CONFIG); // optional
-export const eventMode: Nullable<string> = undefined; // "2023fall";
+export const eventMode: Nullable<string> = "2023fall"; // "2023fall";
```

Comment:
아직 이벤트 기간이 아니니 아직 `eventMode` config value는 설정되면 안되요 ㅜ
```suggestion
export const eventMode: Nullable<string> = undefined; // "2023fall";
```

Reply:
헉 다시 돌려놓았습니다